web/signup.jsp

```diff
@@ -9,9 +9,55 @@
 <html>
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <title>JSP Page</title>
+        <title>Sign-up Page</title>
     </head>
     <body>
-        <h1>Hello World!</h1>
+        <h1>Welcome to My Twitter Website!</h1>
+        <p>
+            Please sign up and then feel free to look around that site! :)
+        </p>
+        <jsp:useBean id="user" scope="session" class="business.User"/>    
+        <form action="signup" method="post">
+            <input type="hidden" name="action" value="add">
+            <label class="pad_top">Full Name:</label>
+            <input type="text" name="user" value="${user.fullName}"><br>
+            <label class="pad_top">Email:</label>
+            <input type="email" name="user" value="${user.emailAddress}"><br>
+            <label class="pad_top">Nickname:</label>
+            <input type="text" name="user" value="${user.nickname}"><br>
+            <label class="pad_top">Password:</label>
+            <input type="text" name="user" value="${user.password}"><br>
+            <select>
+                <option>Month</option>
+                <%
+                    for(int i = 1; i <= 12; i++) {
+                %>
+                <option value=${i}><%= i+"" %></option>
```

Comment:
@GWhite24 I'm not sure if this value is going to work, This commit was just to get the view working. It doesn't look the best but the content is here